source/plugin/project/gradle.rst

```diff
@@ -23,7 +23,7 @@ you have chosen before.**
 .. code-block:: groovy
 
     plugins {
-        `java-library`
+        'java-library'
```

Comment:
I thought it was meant to be backticks?

Reply:
Oh, I see, you got a warning - can it just be escaped though? We must not provide incorrect code, which I think this would be.

Reply:
in Groovy buildscript, this would be:

```groovy
plugins {
  id 'java-library'
}
```

The example with backticks is only correct for Kotlin buildscripts:

```kotlin
plugins {
  `java-library`
}
```

If you wanted an example that would work in either, something like:

```gradle
plugins {
  id("java-library")
}
```

would work, but is not idiomatic for either language.

Reply:
Right - while the code block is marked as "groovy", the block written here is kotlin anyway - especially as it seems to have been copied from the template as noted in the description.

That said, I think this page needs a broader update than just updating the code examples though - for example, it talks about autofilling the plugin name and version, not sure it's true for the former at least. We shouldn't mix kotlin and groovy either which seems to be happening here - choose one or the other (or both), though I would also consider pointing at the template repo.

Reply:
Change to kotlin and added a tip about the template repository.

I've also setup a project using the template repo and made sure the version/description are being autofilled in the sponge_plugins.json

Reply:
Far better, thanks!